tests/test_fitresult.py

```diff
@@ -49,6 +51,9 @@ def create_fitresult(minimizer_class_and_kwargs):
     minimizer = minimizer_class(**minimizer_kwargs)
 
     result = minimizer.minimize(loss=loss)
+    print(result)
+    result.status
+    result.info
```

Comment:
```suggestion
```
Remove that I assume?

Reply:
Haha yes It can be removed if you are not nervous about coverage score :P. Indeed printing and those attributes are never checked and thus lowers the coverage of fit_result.py.

Reply:
:D I see, so I would rather put it somewhere else then and do a check actually. But yeah, pushing the coverage would be nice actually

Reply:
Yeah I agree with that.